src/test/kotlin/com/jetbrains/snakecharm/SnakemakeResolveTest.kt

```diff
@@ -31,7 +31,10 @@ class SnakemakeResolveTest : SnakemakeResolveTestCase() {
     }
 
     fun testExpandTopLevel() {
-        //val result = resolve()
+        assertResolvesTo(PyFunction::class.java, "expand")
+    }
+
+    fun testExpandNested() {
         assertResolvesTo(PyFunction::class.java, "expand")
     }
```

Comment:
Please add more tests, like you have for completion. E.g.
* expand could be resolved out of `rule` block scope, just some top-level python code in file
* expand won't be resolved in not snakemake *.py file

Reply:
Done